sbt_common/storage/src/test/scala/uk/ac/wellcome/storage/test/fixtures/LocalDynamoDb.scala

```diff
@@ -73,4 +76,25 @@ trait LocalDynamoDb extends Eventually with ExtendedPatience {
       .asScala
       .foreach(dynamoDbClient.deleteTable)
   }
+
+  def givenDynamoDbHasItem[T: DynamoFormat](item: T, table: Table) = {
+    Scanamo.put(dynamoDbClient)(table.name)(item)
+  }
+
+  def assertDynamoDbHasNoItems[T: DynamoFormat](table: Table) = {
+    val records = Scanamo.scan[T](dynamoDbClient)(table.name)
+    records.size shouldBe 0
+  }
+
+  def assertDynamoDbHasItem[T: DynamoFormat](id: String, item: T, table: Table) = {
+    val actualRecord = Scanamo.get[T](dynamoDbClient)(
+      table.name)('id -> id)
+    actualRecord shouldBe Some(Right(item))
+  }
+
+  def assertDynamoDbOnlyHasItem[T: DynamoFormat](item: T, table: Table) = {
+    val records = Scanamo.scan[T](dynamoDbClient)(table.name)
+    records.size shouldBe 1
+    records.head shouldBe Right(item)
+  }
```

Comment:
minor: the convention in the SQS tests is to refer to the resource, not the service

In line with that convention, these methods would be better named `givenTableHasItem`, `assertTableEmpty`, `assertTableHasItem`, `assertTableOnlyHasItem`.